libs/accounts/recovery-phone/src/lib/recovery-phone.service.ts

```diff
@@ -39,10 +39,6 @@ export class RecoveryPhoneService {
    * @param region
    */
   public async available(uid: string, region: string): Promise<boolean> {
-    if (!this.config.enabled) {
```

Comment:
Shouldn't we keep this config check? I would expect the available endpoint to return false if the service is turned off.

Reply:
This makes no call to the twilio api, so the check was removed. 

I'm fine with adding this back in. One question though. If this check is added back in and the config is disabled is in a disabled state, i think the `account.recoveryPhone.available` flag will always be false. In this case, would a user be able to delete their recovery phone number?

Reply:
We only use the "available" endpoint to determine if a user can add a recovery phone (based on feature availability and user geolocation) - and it would be expected that this returns false if the service is disabled.

The option to remove is available for any user that has a recovery phone configured (no check on "available").

Reply:
Alright. I'll revert this part then.